frontend/src/pages/CalendarPage/index.js

```diff
@@ -101,6 +104,25 @@ class CalendarPage extends React.Component {
     });
   }
 
+  handleUpdateStatus = async (
```

Comment:
You need not pass in the event object here. Notice what you're passing down to EventCard's event prop, you're passing down selectedEvent already. This means you really only need to pass in the updated status to this handleUpdateStatus() and create a modified selectedEvent from the original selectedEvent whose status is changed to the new status. Think carefully about what you're passing down to functions since you might not need them at all to do what you want.